frontends/concrete-python/concrete/fhe/compilation/keys.py

```diff
@@ -185,20 +185,20 @@ def serialize(self) -> bytes:
         return serialized_keyset
 
     @staticmethod
-    def deserialize(serialized_keys: bytes) -> "Keys":
```

Comment:
we still need both API at this level, so let introduce a new method or an "overload"